lib/jnpr/junos/factcache.py

```diff
@@ -192,7 +193,7 @@ def __len__(self):
         """
         return len(self._callbacks)
 
-    def __repr__(self):
+    def __str__(self):
```

Comment:
Based on these definitions, I believe these two function definitions should be reversed.
`__str__()` should be the formatted and nicely printable representation.
https://docs.python.org/2/library/functions.html#str
`__repr__()` should be the string which could be passed back to `eval()`
https://docs.python.org/2/library/functions.html#repr